tests/checkpoint/test_checkpoint.py

```diff
@@ -683,6 +779,124 @@ def test_checkpoint_configuration_using_RuntimeDataConnector_with_Airflow_test_y
     assert len(data_context.validations_store.list_keys()) == 1
     assert result.success
 
+    assert mock_emit.call_count == 6
+
+    # noinspection PyUnresolvedReferences
+    expected_events: List[unittest.mock._Call] = [
+        mock.call(
+            {
+                "event": "data_context.test_yaml_config",
+                "event_payload": {
+                    "anonymized_name": "f563d9aa1604e16099bb7dff7b203319",
+                    "parent_class": "Checkpoint",
+                },
+                "success": True,
+            },
+        ),
+        mock.call(
+            {
+                "event": "data_context.get_batch_list",
+                "event_payload": {
+                    "anonymized_batch_request_required_top_level_properties": {
+                        "anonymized_datasource_name": "a732a247720783a5931fa7c4606403c2",
+                        "anonymized_data_connector_name": "d52d7bff3226a7f94dd3510c1040de78",
+                        "anonymized_data_asset_name": "556e8c06239d09fc66f424eabb9ca491",
+                    },
+                    "batch_request_optional_top_level_keys": [
+                        "batch_identifiers",
+                        "runtime_parameters",
+                    ],
+                    "runtime_parameters_keys": ["batch_data"],
+                },
+                "success": True,
+            },
+        ),
+        mock.call(
+            {
+                "event": "data_asset.validate",
+                "event_payload": {
+                    "anonymized_batch_kwarg_keys": [],
+                    "anonymized_expectation_suite_name": "__not_found__",
```

Comment:
Why are these payload items `"__not_found__"`?

Reply:
@anthonyburdi I am not sure -- this test method existed before (the current PR did not introduce any change other than add the test for events for this method with the focus on the content of the "data_context.get_batch_list" and "checkpoint.run" events, not the others before them).

Reply:
Makes sense - also this part of the message is for an old event: `data_asset.validate`